qa/L0_cmdline_trace/test.sh

```diff
@@ -576,4 +570,59 @@ else
 fi
 
 
+# check deprecation warnings 
```

Comment:
Can be follow-up, but add some negative tests and other config setting tests. Some off the top of my head:

1. Both triton and opentelemetry modes cannot both be specified at same time (I think?)
2. Can't set tensor level for opentelemetry mode
3. Assert default mode is Triton when not specifying any mode